p2p/protocol/circuitv2/client/transport.go

```diff
@@ -46,8 +46,16 @@ func AddTransport(h host.Host, upgrader transport.Upgrader) error {
 
 // Transport interface
 var _ transport.Transport = (*Client)(nil)
+var _ transport.SkipResolver = (*Client)(nil)
```

Comment:
Also similar documentation here, so that in case some one does actually embed this, they remember to wrap the `SkipResolver` interface too.